tests/tests/son_wallet_tests.cpp

```diff
@@ -0,0 +1,114 @@
+#include <boost/test/unit_test.hpp>
+
+#include "../common/database_fixture.hpp"
+
+#include <graphene/chain/hardfork.hpp>
+#include <graphene/chain/son_wallet_object.hpp>
+#include <graphene/peerplays_sidechain/defs.hpp>
+
+using namespace graphene::chain;
+using namespace graphene::chain::test;
+
+BOOST_FIXTURE_TEST_SUITE( son_wallet_tests, database_fixture )
+
+BOOST_AUTO_TEST_CASE( son_wallet_create_test ) {
+
+   BOOST_TEST_MESSAGE("son_wallet_create_test");
+
+   generate_block();
+   set_expiration(db, trx);
+
+   ACTORS((alice));
+
+   generate_block();
+   set_expiration(db, trx);
+
+   {
+      BOOST_TEST_MESSAGE("Send son_wallet_create_operation");
+
+      son_wallet_create_operation op;
+
+      op.payer = db.get_global_properties().parameters.get_son_btc_account_id();
```

Comment:
Just curious, isn't this failing as it will be GRAPHENE_NULL_ACCOUNT if no active SONs are present?

Reply:
Evaluator check

FC_ASSERT(db().get_global_properties().parameters.get_son_btc_account_id() != GRAPHENE_NULL_ACCOUNT, "SON paying account not set.");

is disabled, in order to make tests pass.

For some reason, generating maintenance block in tests does not properly trigger creation of son_btc_account. However, this is not important for testing basic functionality of the son wallet operations.